Translation/en/Text/Main/Tutorial-Communication.txt

```diff
@@ -0,0 +1,69 @@
+#set level -1024
+会話やスキンシップで仲良くなろう！=Get close via conversations and skinship!
+コミュニケーション：会話や一緒に行動することで好感度が変化します。= Communication: Talking/doing activities together raises a girl's affection.
+●話を振る=● Talking
+「世間話」<size%3D30>や</size>\n「コイバナ」<size%3D30>など</size>\n  話を振ると=Have\n“Small Talk”,\n“Love Talk” etc, and...
+「好感度」=Her\nAffection
+<size%3D30>が</size>上がります=\n       goes up!
+spacing to align with above text
+●スキンシップ=● Skinship
+「触る」=    Caress
+「見る」=        Stare
+//spacing shit to match picture
+スキンシップで=\nUse skinship
+女の子<size%3D36>の</size>反応=      and enjoy
+//spacing shit to align with line below
+<size%3D30>を</size>楽しもう！=her reactions!
+●コイカツ=● Koikatsu
+女の子と=Do things
+一緒<size%3D36>に</size>行動=together with a girl
+島巡り=Travel
+<size%3D24>好感度</size>「愛の絆」<size%3D24>の時</size>\n特別なデートスポット\n<size%3D24>が</size>出現<size%3D24>します。</size>=Special date spots appear if you share a “true love”
+「話題」を使い、女の子とのコミュニケーションをより楽しもう！=  Use “topics” to get more out of your conversations!
+//spacing shit
+「話題」の獲得=  Getting topics
+マップで拾ったり=From spots
+話を振ったり=  By talking
+一緒に\n行動したり\n<size%3D30>することで</size>=Doing\njoint activities
+<size%3D36>が</size>手に入る !!=are used in →
+//Ignore line with white space
+●獲得した話題を使って「聞く」、「話題を振る」=● Using topics when replying/asking
+話題のカテゴリー=Topic category
+「名物」=Specialty
+「レジャー」=   Leisure
+「自然」=Nature
+「海」=Sea
+<size%3D30>女の子から</size>振られた話題<size%3D30>に</size>=When replying to a girl
+<size%3D30>合わせたり…</size>=\nReply with a\nmatching topic
+<size%3D30>「話題を振る」から</size>\n  高レアの話題\n<size%3D30>  を使ったりして</size>= Use a rare topic\n when asking
+//spacing shit
+ヤシガニ=☆ 3
+ウミガメ=☆ 4
+//these two lines can never be aligned properly holy fuck
+※☆３以上は１度だけ！=※ ☆ 3 or more topics\ncan only be used once!

```

Comment:
How does this work in practice? Once per conversation? Once per girl? Not once ever, right?

Reply:
Once per girl, doesn't reset if you pass the time or advance to the next day.

Reply:
Can that be fit in there somehow?
```suggestion
※☆３以上は１度だけ！=※  ☆ 3+ topics can only be\nused once per girl!

```

Reply:
Yep, it fits! I'll put updated screenshots in a separate comment